.buildkite/commands/release-upload.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -eu
 
+BETA_RELEASE=${1:-true} # use first call param, default to true for safety
+
+echo "Running $0 with BETA_RELEASE = $BETA_RELEASE..."
```

Comment:
I thought moving the code that reads the call parameters to the start of the script would make the existence and requirement of those parameters clearer.

An alternative would be to remove the fallback value and fail if `$1` is missing. I thought of that option only now, while writing this comment, but I like it more. Better to fail early, IMHO.

I'll leave the PR as is for the moment. If the fail early idea is well received, I'll create a task to roll it out consistently across our repos.

Reply:
tbh I don't mind either way. Though we've been using in other repos the validation as in `MY_VAR="${1?Error message.}"`, which can also be a convention to follow here.